.travis.yml

```diff
@@ -8,8 +8,7 @@
 language: python
 sudo: false
 cache:
-  directories:
-    - $HOME/.cache/pip
+  - pip
```

Comment:
We needed the former because we override the `install` step.  http://docs.travis-ci.com/user/caching/#pip-cache